stack-windows.yaml

```diff
@@ -1,32 +0,0 @@
-resolver: nightly-2020-06-19
```

Comment:
Same with this file, i would like to make it work.

Reply:
Use git revert